
# Motility and occurrence of fly-associated microbes

This repository provides supporting information and the codes for the following manuscript. <br />
 <br />

## Motility genes are associated with the occurrence of *Drosophila melanogaster*-associated gut microbes

Rishi Bhandari<sup>a</sup>, Caleb J. Robbins<sup>b</sup>, , Arinder K. Arora<sup>c</sup>, John M. Chaston<sup>*d</sup>, David S. Kang <sup>a</sup>  <br />
 <br />
 <br />
<sup>a</sup> USDA Agricultural Research Service, Biological Control of Insects Research Laboratory, Research Park, 1503 S Providence, Columbia, MO 65203-3535, USA

<sup>b</sup> Center for Reservoir and Aquatic Systems Research, Baylor University, Waco, TX 76798, USA

<sup>c</sup> Department of Entomology, Texas A&M, 2301 Experiment Station Rd, Bushland, TX 79012

<sup>d</sup> Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah 84604, USA <br />
 <br />
 <br />       

Short title: Impacts of host crop and fly sex on the Drosophila suzukii microbiome

Corresponding Author: Dave Kang, Dave.Kang@usda.gov



    
   ### *Note: This study is currently under review.*   <br />  
  <br />  

  
## Abstract
Recent work highlighted the role of motility genes in dispersing fly-associated microbes and their spread between hosts. We investigated whether bacterial genes encoding motility are associated with the occurrence of bacteria above passive dispersal levels in the gut of wild Drosophila melanogaster. We revisited 16S amplicon and shotgun metagenome data of wild flies and correlated four genera of bacteria (Commensalibacter, Gluconobacter, Lactobacillus, and Tatumella) with motility genes. We plotted the microbes against neutral models of ecological drift and passive dispersal. Microbes with positive correlations to motility were exclusively found at or above neutral model predictions, suggesting motility genes are crucial for fly microbiota spread and colonization. This information is crucial for understanding how specific gene functions contribute to microbial community dispersal and colonization within the fly host. Moreover, this study’s findings serve as a proof of concept for using the neutral model to predict microbial functions essential for survival and dissemination in diverse hosts.

### Keywords: Drosophila melanogaster, gut microbiome, neutral model, dispersal, bacteria


## Data Availability

The data examined herein were obtained from a previous study of microbial metagenomics and fly lipid content entitled “Functional traits of the gut microbiome correlated with host lipid content in a natural population of Drosophila melanogaster” (https://doi.org/10.1098/rsbl.2019.0803). All the data from the study are available as electronic supplementary material or at NCBI Short Read Archive under PRJNA579293


## Acknowledgements
We thank the USDA-ARS Biological Control of Insects Research Laboratory (BCIRL) and all the members of the insect microbiome laboratory who contributed ideas and suggestions to this project.

## Funding
This research was funded by the foundation grant GRANT12216941 from the National Institute of Food and Agriculture.
Rishi Bhandari was supported by a postdoctoral fellowship funded by the USDA Agricultural Research Service's SCINet Program and AI Center of Excellence, ARS project numbers 0201-88888-003-000D and 0201-88888-002-000D, and administered by the Oak Ridge Institute for Science and Education (ORISE) through an interagency agreement between the U.S. Department of Energy (DOE) and the U.S. Department of Agriculture (USDA). ORISE is managed by ORAU under DOE contract number DESC0014664. All opinions expressed in this paper are the authors and do not necessarily reflect the policies and views of the USDA, DOE, or ORAU/ORISE.

